src/client/javascripts/App.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import {
+    AppRegistry,
+    StyleSheet,
+    Text,
+    View,
+    Platform
+} from 'react-native';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        titleText: 'Yoda',
+        os: 'OS',
+      };
+
+    //For specific platform
```

Comment:
I think we need 1 space after // and before For.
So it'd read:
// For specific platform

Also, this comment is pretty bad (not clear what this comment means). Either remove the comment, or leave a more descriptive comment like:

// Populate the OS state by looking at Platform params.